src/constants.ts

```diff
@@ -104,6 +108,15 @@ export enum Decision {
 	PROPER_REPACK_MISMATCH = "PROPER_REPACK_MISMATCH",
 	RESOLUTION_MISMATCH = "RESOLUTION_MISMATCH",
 }
+export type DecisionAllMatch =
+	| Decision.MATCH
+	| Decision.MATCH_SIZE_ONLY
+	| Decision.MATCH_PARTIAL;
+export const DECISION_ALL_MATCH = [
+	Decision.MATCH,
+	Decision.MATCH_SIZE_ONLY,
+	Decision.MATCH_PARTIAL,
+];
```

Comment:
A more idiomatic way to express this in TypeScript would be to write a type predicate:
```ts
export function isDecisionAllMatch(decision: Decision): decision is DecisionAllMatch {
  return decision === Decision.MATCH || 
    decision === Decision.MATCH_SIZE_ONLY || 
    decision === Decision.MATCH_PARTIAL
}
```
`DECISION_ALL_MATCH.includes(decision)` can be replaced with `isDecisionAllMatch(decision)` and we won't lose type fidelity at the call site.